### PR TITLE
Link to see ships and sites that can spawn in current sector

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image

  - rscadd: "Added link to see sites and ships that can spawn in current sector, shown near the round start."

Why:
- It is not easy to check this in the code, except at runtime with a debugger. There is no simple list to just check all sites/ships that spawn in some sector.
- I sometimes see people wondering if X or Y ship can spawn in current sector, where they are not sure if it's cause X or Y is broken or it's a bug, or if it's just bad luck that the thing didn't spawn lately.

### CODE REVIEW

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — COMMENTED** [review]
Have to increase TO_WORLD_COUNT to 186 in .github\workflows\byond.yml for the tests to run

File: code/controllers/subsystems/ticker.dm [hunk 1]
```diff
@@ -464,6 +464,20 @@ var/datum/controller/subsystem/ticker/SSticker
 			+ "<br>" \
 		)
 
+	var/datum/space_sector/current_sector = SSatlas.current_sector
+	var/html = {"\
+		<span class='notice'>\
+			Current sector: [current_sector]. \
+		</span>\
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/controllers/subsystems/ticker.dm]: Addressed

**Comment by DreamySkrell** [comment]
!review

File: code/modules/background/space_sectors/space_sector.dm [hunk 2]
```diff
@@ -115,4 +115,13 @@
 	return "[rand(1, 3)] days"
 
 /datum/space_sector/proc/generate_system_name()
-	return "[pick("Miranda", "BNM", "Xavier", "GJ", "HD", "TC", "Melissa", "TC")][prob(10) ? " Eridani" : ""] [rand(100,999)][prob(10) ? " [pick(greek_letters)]" : ""]"
\ No newline at end of file
+	return "[pick("Miranda", "BNM", "Xavier", "GJ", "HD", "TC", "Melissa", "TC")][prob(10) ? " Eridani" : ""] [rand(100,999)][prob(10) ? " [pick(greek_letters)]" : ""]"
+
+// Returns a flat list of all possible away sites that can spawn in this sector.
```
Comment (FluffyGhoster) [inline code/modules/background/space_sectors/space_sector.dm:120]:
```suggestion
/// Returns a flat list of all possible away sites that can spawn in this sector.
```

**Review by FluffyGhoster — APPROVED** [review]

File: code/controllers/subsystems/ticker.dm [hunk 3]
```diff
@@ -464,6 +464,15 @@ var/datum/controller/subsystem/ticker/SSticker
 			+ "<br>" \
 		)
 
+	var/datum/space_sector/current_sector = SSatlas.current_sector
+	var/html = SPAN_NOTICE("Current sector: [current_sector].") + {"\
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/controllers/subsystems/ticker.dm:468]: image

There's a space in the next line there: `<span> \`

[the remark this answers is not in the document]
Comment (FluffyGhoster) [reply code/controllers/subsystems/ticker.dm:468]: Perfect, that works

**Pull request merged**